### ISSUE TITLE
String Interpolation incorrectly quoted when used as parent selector

### ISSUE THREAD

**Issue by petebrowne** [sass/libsass#1770]
Input SCSS:

``` scss
@function returns-string() {
  @return "selector";
}

#{"selector"} {
  color: red;
}

#{returns-string()} {
  color: red;
}

#{"selector"} selector2 {
  color: red;
}

#{returns-string()} selector2 {
  color: red;
}
```

Ruby Sass output:

``` css
selector {
  color: red; }

selector {
  color: red; }

selector selector2 {
  color: red; }

selector selector2 {
  color: red; }
```

Libsass (via node-sass)

``` css
selector {
  color: red; }

selector {
  color: red; }

selector selector2 {
  color: red; }

"selector" selector2 {
  color: red; }
```

**Comment by xzyfer** [comment]
Thanks for the report @petebrowne. I can confirm this issue is present on master.

**Comment by xzyfer** [comment]
Spec added https://github.com/sass/sass-spec/pull/638

**Comment by xzyfer** [comment]
This is fixed and will be in the upcoming 3.3.3. Subscribe to #1808 for future updates on that release.
